### ISSUE TITLE
Psychiatrist's office privacy

### ISSUE THREAD

**Issue by ZippyShanks** [VOREStation/VOREStation#1398]
If I'm to guarantee the privacy of my appointments, I would like it to be made so that people can't just wander into the psychiatrist's office or that at the very least I can lock my door when I am dealing with someone's issues.

I would very much like to see a dormroom style lock on the door to the psychiatrist's office. It has happened more than once now that a curious EMT or even the CMO has entered my office mid session, it has even occurred that K9 borgs have used my office to chase eachother while I was in session with a client. And, if at all possible, I would like the door between the Psychiatrist's closet and the morgue removed, if not at least being able to be locked from the inside in case it could be used as an emergency escape.  

In short, what I'd like to see happen is:
A button to lock both exits to the Psychiatrist's office from the inside.
(optional) The removal of the door between the psychiatrist's office and the morgue.

**Comment by Screemonster** [comment]
The old psych office had no back door and also privacy tint on the windows, so this makes sense.

**Comment by Wickedtemp** [comment]
Window tint and maybe a holosign on the door that says "In Session" should suffice. 

I'd disagree with the lock because, well, you'd be locking the CMO out of an area of their own department. If you have a CMO that won't leave you alone and they know you're in session, you could report it and have them dealt with. 

I've had privacy issues with the "Counselor"s office and Psych as well in the past, mainly when the AI decided to let the chef run around the office and loot it WHILE THE COUNSELOR WAS IN SESSION, windows tinted and all. However, there's also been times where a psychiatrist assaulted and hit their patient and the CMO had to intervene, so I'd be against a lock. 

Psych/Counselor appointments are extremely sensitive information, I personally don't even like that the CD has access to medical comms, allowing them to listen in on matters involving these appointments, but I know they won't lose that bit of access anytime soon.

**Comment by ZippyShanks** [comment]
Still, I would argue for it because I would very much like to be able to guarantee that privacy, at least on the front door. If you really want the CMO to be able to intervene, then you could of course only put the lock on the front door and allow the morgue opening to persist as a kind of intervention or emergency exit. 

I don't mind the other issues, such as that the reports I write can be read, as I send most of them to Central regardless when they have to do with performance evaluation or such issues. My earlier requests for safe storage of sensitive information have already been allowed and acted upon. Now I just want to be able to prevent people from lolrandoming into the place. If an in session sign above the door can help, then I'll gladly make that concession if a full lock is too much to ask.

**Comment by Wickedtemp** [comment]
And if I remember correctly, Doctors, Chemists, and Paramedics don't even have access to the office. Or at least, doctors don't, I'm pretty sure. 

I suggest removing the walls, replace them with windows, so if there's an appointment those windows can be tinted, add an 'In Session' holosign to the airlock itself, making it extremely obvious that there's an ongoing appointment. As a CMO main, I consider it part of policy, as well as just plain courteous, to ASK FIRST before just barging in, and if no reply is made within ten minutes, then it's one of those "I respected your privacy by knocking, but now I'm exercising my authority as the CMO by coming in because you didn't answer" moments. Same with tinted subacutes. Most of the time those are vacant anyways.

**Comment by Dhaeleena** [comment]
Should no longer be relevant on the current map. Place is pretty private, has the proper access on the doors and tons of windows you can just tint for privacy.
